### PR TITLE
Fix player in title screen jumping slopes

### PR DESCRIPTION
**Pull request by MatusGuy**

### CODE REVIEW

File: src/supertux/title_screen.cpp [hunk 1]
```diff
@@ -226,12 +228,25 @@ TitleScreen::update_level(float dt_sec)
   m_controller->press(Control::RIGHT);
 
   // Check if we should press the jump button
-  Rectf lookahead = player.get_bbox();
-  lookahead.set_right(lookahead.get_right() + 96.f);
-  lookahead.set_bottom(lookahead.get_bottom() - 2.f);
+  Vector eye = player.get_bbox().get_middle();
+  eye.x = player.get_bbox().get_right();
```
[the remark this answers is not in the document]
Comment (Vankata453) [reply src/supertux/title_screen.cpp:232]: `bbox.get_top() + bbox.get_height()` is the same as `bbox.get_bottom()`.

[the remark this answers is not in the document]
Comment (Vankata453) [reply src/supertux/title_screen.cpp:232]: Oops, nevermind!

**Review by Vankata453 — APPROVED** [review]
Works good, should be ready to go after mrkubax10's suggestion is implemented.

File: src/supertux/title_screen.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (tobbi) [reply src/supertux/title_screen.cpp:232]: Please add spaces around the `/` operator. I'm being nitpicky like that.

**Review by mrkubax10 — APPROVED** [review]

**Pull request merged**